lib/chef/provider/package/chocolatey.rb

```diff
@@ -130,17 +130,21 @@ def remove_package(names, versions)
         # install from, but like the rubygem provider's sources which are more like repos.
         def check_resource_semantics!; end
 
-        def get_choco_version
+        def self.get_choco_version
           @get_choco_version ||= powershell_exec!("choco --version").result
         end
 
         # Choco V2 uses 'Search' for remote repositories and 'List' for local packages
-        def query_command
+        def self.query_command
           return "list" if get_choco_version.match?(/^1/)
 
           "search"
```

Comment:
Isn't it still using `list` for v1 and `search` for v2 contrary to pr description?